deploy/kubernetes/README.adoc

```diff
@@ -227,7 +213,7 @@ Now that the application has been deployed and verified, it can be registered to
 The Open Service Broker API endpoints in the service broker application are secured with a basic auth username and password. Create a Kubernetes secret to store these credentials:
 
 ----
-$ kubectl create -f deploy/kubernetes/service-broker-secret.yml
+$ kubectl create secret generic -n bookstore-broker bookstore-broker-secret --from-literal=username='admin' --from-literal=password='supersecret'
```

Comment:
I like the consistency of creating all artifacts from files, but I'm OK with creating with CLI options also. This commit should remove `deploy/kubernetes/service-broker-secret.yml` if it's not going to be used though.